### DISCUSSION TITLE
Crash single-instanced app when drag and drop

### DISCUSSION THREAD

**Discussion by Bondarenko1990** [microsoft/WindowsAppSDK#3936] [Q&A]
I remade the application according to this example [Making the app single-instanced (Part 3)](https://blogs.windows.com/windowsdeveloper/2022/01/28/making-the-app-single-instanced-part-3/)
And when dragging the grid control I get a crash:
![image](https://github.com/microsoft/WindowsAppSDK/assets/32612104/0bf449bd-fe90-41ea-8c13-8596cb4aef50)
Event Viewer logs:
![image](https://github.com/microsoft/WindowsAppSDK/assets/32612104/1096ac9b-f279-4e68-818c-8432e9aa0ae5)

**Comment by bpulliam** [comment]
Tracked with issue.

**Reply by Juansero29** [reply]
Issue: https://github.com/microsoft/microsoft-ui-xaml/issues/9061
